fastergt/src/play/modules/gtengineplugin/gt_integration/GTFileResolver1xImpl.java

```diff
@@ -87,7 +87,7 @@ public GTTemplateLocationReal getTemplateLocationFromRelativePath(String relativ
     URL url = null;
     if (vf == null) {
       try {
-        final String classloadedPrefix = "/" + Play.tmpDir.getName() + "/";
+        final String classloadedPrefix = "/" + Play.tmpDir.getName() + "/" + TemplateLoader.CLASSPATH_LOADED_TEMPLATE_TMP_PATH_PREFIX;
```

Comment:
This dies with NPE when `play.tmpDir=none` is set in the `application.conf`, but a fix is ready, just a few tests need to be cooked.